api/tokens.go

```diff
@@ -162,6 +183,15 @@ func (api *API) SignOutHandler(ctx context.Context) http.HandlerFunc {
 	}
 }
 
+func adminUserGlobalSignOut(authParams AuthParams, userPoolId string, cognitoClient cognito.Client) (err error) {
+	adminUserGlobalSignOutInput := buildAdminSignoutRequest(authParams, userPoolId)
+	_, err = cognitoClient.AdminUserGlobalSignOut(adminUserGlobalSignOutInput)
```

Comment:
Is err the only indicator this action failed? For example executing the http request was successfully but the response returned contained a non success status code. Does that return an error or do you have to check the other return var to check if it was successful?

Reply:
This is a really good question, because it wasn't too clear to me from the documentation: https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminUserGlobalSignOut.html#API_AdminUserGlobalSignOut_ResponseElements

If possible itt would be really good to have chat about this

Reply:
The doc isn't clear to me but I think its saying that it will return an `awserr.Error` for all failures. So what you have here is fine based on that assumption/understanding.